nav2_bt_navigator/behavior_trees/navigate_w_recovery_and_replanning_only_if_path_becomes_invalid.xml

```diff
@@ -10,8 +10,11 @@
       <PipelineSequence name="NavigateWithReplanning">
         <RateController hz="1.0">
           <RecoveryNode number_of_retries="1" name="ComputePathToPose">
-            <Fallback> 
+            <Fallback>
               <ReactiveSequence>
+                <Inverter>
+                  <GlobalTimeExpired/>
```

Comment:
I don't understand why this is added

Reply:
To trigger re-planning at a desired frequency.